src/libraries/pkg/test/testPackages.proj

```diff
@@ -177,7 +177,7 @@
           Condition="'$(ArchiveTests)' != 'true'">
 
     <PropertyGroup>
-      <TestRestoreCommand>$(TestDotNetPath)</TestRestoreCommand>
+      <TestRestoreCommand>&quot;$(TestDotNetPath)&quot;</TestRestoreCommand>
```

Comment:
as we are inside a propertygroup you can just use the `"` here.

Reply:
Ok sure, thanks for confirming that. I wasn't sure the character was allowed.

Reply:
```suggestion
      <TestRestoreCommand>"$(TestDotNetPath)"</TestRestoreCommand>
```